### DISCUSSION TITLE
how to get spesific array value from datatypecontent

### DISCUSSION THREAD

**Discussion by hyugihc** [thedevdojo/voyager#5326] [Q&A]
i want to get value of download_link in this json

{"id":2,"nomor_sk":"2","tanggal_sk":"2021-05-06","perihal_sk":"Perihal SK","file_sk":"[{\"download_link\":\"cans\\\\May2021\\\\RAIlfhQnU4vx3LNbX1z1.PNG\",\"original_name\":\"bersih2.PNG\"}]","created_at":"2021-05-09T13:23:52.000000Z","updated_at":"2021-05-11T22:34:38.000000Z","deleted_at":null,"approval":null,"wilayah":null}

on laravel blade if i want to get the perihal_sk value i can call

{{$dataTypeContent->perihal_sk }}

but how to call only download_link value?

because when i call

{{$dataTypeContent->file_sk }}

it will get

[{"download_link":"cans\May2021\RAIlfhQnU4vx3LNbX1z1.PNG","original_name":"bersih2.PNG"}]

while i only want the download_link value or original_name value. the result i want to get is only

"cans\May2021\RAIlfhQnU4vx3LNbX1z1.PNG" or "bersih2.PNG"

**Comment by MrCrayon** [comment]
That's saved in json, you can see how to get that value here:
https://github.com/the-control-group/voyager/blob/d5559d29388188adebfd42d643ee4581c615426a/resources/views/formfields/file.blade.php#L3
